documents/knowledge/elements.yaml

```diff
@@ -1149,6 +1149,8 @@
   arity: 2
   overloads:
     any-any: a.replace(b,"")
+    num-fun: first a positive integers where b is truthy
+    fun-num: first b positive integers where a is truthy
```

Comment:
```suggestion
    num-fun: first a positive integers where b is truthy
    fun-num: first b positive integers where a is truthy
    any-any: a.replace(b,"")
```
It just seems like the any-any overloads is matched before the num-fun/fun-num overloads right now.

Reply:
What do you think of a file containing the order of overloads?

Reply:
`elements.yaml` has that already